packages/shared/src/utilities/color/isDark.ts

```diff
@@ -10,7 +10,14 @@ import { toShortRgba } from './toShortRgba';
  * @param {Object} Object of RGBA values
  * @returns {Boolean} Return if the color is dark
  */
-export const isDark = (color: Color): boolean => {
-    const parsedColor = tinycolor(toShortRgba(color));
+
+const isRgbaLongFormat = (value: Color) => {
+    const requiredKeys = ['red', 'green', 'blue'];
+    return typeof value === 'object' && requiredKeys.every((i) => value.hasOwnProperty(i));
+};
+
+export const isDark = (color: unknown): boolean => {
+    const inputColor = isRgbaLongFormat(color as Color) ? toShortRgba(color) : (color as ColorInput);
+    const parsedColor = tinycolor(inputColor);
     return parsedColor.isDark() || (parsedColor.getAlpha() > 0.25 && parsedColor.getAlpha() < 1);
```

Comment:
Just FYI: there is a function in the tinycolor library to get the most readable color: https://www.npmjs.com/package/tinycolor2#mostreadable

Reply:
Nice! But I'd rather keep it as is, because `isDark` now returns a boolean and `mostReadable` returns you the most readable color value. Don't see a benefit in using that.